src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carPrice = 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = days * carPrice - 50;
+
+    return result;
+  }
+
+  if (days >= 3) {
+    result = days * carPrice - 20;
+
+    return result;
```

Comment:
The `return result;` statement is repeated here as well. You can optimize the function by moving the return statement outside the conditional blocks.